core/src/main/java/org/apache/carbondata/core/datastore/page/statistics/SimpleStatsResult.java

```diff
@@ -25,6 +25,8 @@
 
   Object getMax();
 
+  byte getNull();
```

Comment:
Better use boolean and change method to isNull

Reply:
SimpleStatsResult is the statistics for a ColumnPage, what does getNull() means for a page?

Reply:
We can do it. But in later phase of execution the Boolean will again get converted to Byte[]. So it doesn't make that much difference.